components/infiniteLogoCarousel.tsx

```diff
@@ -3,7 +3,7 @@ import { InfiniteMovingCards } from "~/components/ui/infinite-moving-cards";
 export const InfiniteLogoCarousel = () => {
   return (
     <div className=" relative flex flex-col items-center justify-center overflow-hidden rounded-md antialiased">
```

Comment:
For formatting, remove the leading space in the className.
![sdnkcnjsnjk](https://github.com/user-attachments/assets/72d60934-4d82-4383-bd64-b1296de83736)

Reply:
Is this resolved?

Reply:
no not yet